mne/report.py

```diff
@@ -800,7 +811,7 @@ def _add_figs_to_section(self, figs, captions, section='custom',
                                              caption=caption,
                                              show=True,
                                              image_format=image_format,
-                                             width=scale, comment=comment)
```

Comment:
I seem to get the correct output with this line omitted (scale up by 2 for scale=2, down by factor of 2 for scale=0.5)